test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -2146,6 +2146,80 @@ public void NonDefaultColNamesMultiClassificationOnnxConversionTest()
             Done();
         }
 
+        [Fact]
+        public void PcaOnnxConversionWithCustomOpsetVersionTest()
+        {
```

Comment:
This test is pretty similar to `PcaOnnxConversion`. Can we remove this new test and modify the original with [InlineData(9)], [InlineData(10)], [InlineData(11)] attributes and `public void PcaOnnxConversionTest(int opsetVersion)`?